### DISCUSSION TITLE
is there a way

### DISCUSSION THREAD

**Discussion by sussybalsin2020** [3kh0/ext-remover#784] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
board: dedede

blocker extension: iboss 

cant downgrade

sh1mmer doesnt work

unenrollment doesnt work

dev mode is blocked 

webstore is blocked

playstore is blocked

javascript is blocked

crosh dev shell is blocked

version 118

force re-enrollment is enabled

my chromebook is the hp chromebook x360 11 g4 ee

dns is blocked

removing the battery doesnt work

write protection is enabled

is there anything i can do to to un-enroll or enter dev mode on this chromebook

**Comment by AshtonDavies** [comment]
> board: dedede
> 
> blocker extension: iboss 
> 
> cant downgrade
> 
> sh1mmer doesnt work
> 
> unenrollment doesnt work
> 
> dev mode is blocked 
> 
> webstore is blocked
> 
> playstore is blocked
> 
> javascript is blocked
> 
> crosh dev shell is blocked
> 
> version 118
> 
> force re-enrollment is enabled
> 
> my chromebook is the hp chromebook x360 11 g4 ee
> 
> dns is blocked
> 
> removing the battery doesnt work
> 
> write protection is enabled
> 
> is there anything i can do to to un-enroll or enter dev mode on this chromebook
> 
> 
> 
> 

How the heck is DNS blocked? And even if it is, it's technically impossible to really block it.

**Reply by sussybalsin2020** [reply]
on my schools wifi it is inaccesable and when i try to enter any other dns on other wifi's it will reset

**Reply by AshtonDavies** [reply]
His method is kinda better than Sh1mmer.

Also, you can't connect to a DNS if your school has no guest network and you have no hotspot.

**Reply by sussybalsin2020** [reply]
ok

**Comment by S-PScripts** [comment]
stale discussion only applies to one person

'answered' discussion

(maybe use incognito)
